### ISSUE TITLE
DB is missing 

### ISSUE THREAD

**Issue by deepakgarg1983** [ddevilred1/Dental-Clinic-Management-System#2]
Dear ,

Can you share DB file so that we can use this project for demonstration !! 

Thank You
deepakgarg1983@rediff.com

**Comment by muskan975** [comment]
Yes please can u share DB file so that we can use it in project 
Thank you
@muskanpore13@gmail.com
